src/main/java/racingcar/domain/Car.java

```diff
@@ -0,0 +1,15 @@
+package racingcar.domain;
+
+public class Car {
+
+    private final int INCREASE_STANDARD_NUMBER = 4;
+    private int moveCount = 0;
+
+    public int stopOrMove(int randomValue) {
+        if (INCREASE_STANDARD_NUMBER <= randomValue) {
+            moveCount = moveCount + 1;
+            return moveCount;
+        }
+        return moveCount;
+    }
```

Comment:
```suggestion
    public void stopOrMove(int randomValue) {
        if (INCREASE_STANDARD_NUMBER <= randomValue) {
            moveCount += 1;
        }
    }
```
1칸 전진 또는 정지 메서드네요
이럴 때는 moveCount만 증감시키고 
Car가 얼마나 이동되었는지는 getter를 사용하여 처리해도 되지 않을까요?

Reply:
인라인 메서드 형태로 만들고 싶어서 위 방식대로 진행했는데 준혁님께서 말씀해주신 것이 보다 더 깔끔하고 메서드 역할에 명확한 것 같습니다!